src/css/style_two.css

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
can we rename and re-organize this css?  I feel maybe we should put all our login css in a file called login.css, and we can continue this pattern as we go (agent_dashboard.css, traveler_dashboard.css, new_trip.css), etc

Reply:
Removed the CSS file I wasn't using and renamed the relevant CSS login.css.